src/signals/incident-management/components/CheckboxList/doc/README.md

```diff
@@ -0,0 +1,193 @@
+# CheckboxList
+
+Component that renders a list of checkboxes with an optional toggle to check or uncheck all boxes in the group at once.
+
+## Simple group
+
+```jsx
+<CheckboxList
+  title="Stadsdeel"
+  name="fooBarBaz"
+  options={[
+    { key: 'A', value: 'Centrum' },
+    { key: 'B', value: 'Westpoort' },
+  ]}
+/>
+```
+
+will render:
+
+<img src="./simple.png" alt="" width="120" />
+
+HTML:
+
+```html
+<input data-id="A" id="fooBarBaz_A" name="fooBarBaz" type="checkbox" value="A">
+<input data-id="A" id="fooBarBaz_A" name="fooBarBaz" type="checkbox" value="B">
+```
+
+## Group with toggle
+
+Any group can be given the option to render a toggle that checks or unchecks all boxes at once.
+
+When a toggle is rendered, the component's behaviour is as follows:
+- clicking the toggle will check or uncheck all boxes
+- the toggle label text will be the value of prop
+  - `toggleAllLabel`: when NOT having toggled all boxes or NOT having checked all boxes manually
+  - `toggleNothingLabel`: when having toggled all boxes or having checked all boxes manually
+
+```jsx
+<CheckboxList
+  hasToggle
+  name="fooBarBaz"
+  options={[
+    { key: 'A', value: 'Centrum' },
+    { key: 'B', value: 'Westpoort' },
+  ]}
+  title="Stadsdeel"
+/>
+```
+
+will render:
+
+<img src="./toggle.png" alt="" width="220" />
+
+HTML:
+
+```html
+<!-- toggle -->
+<label>
+  Alles selecteren
+</label>
+
+<!-- checkboxes -->
+<input data-id="A" id="fooBarBaz_A" name="fooBarBaz" type="checkbox" value="A">
+<input data-id="A" id="fooBarBaz_A" name="fooBarBaz" type="checkbox" value="B">
+```
+
+## Group with named toggle
+
+When a group of checkboxes can be represented by a single value when all boxes are checked, the component can be given a name for the toggle element (which will render a checkbox with that name) which will show up in the submitted form data.
+
+```jsx
+<CheckboxList
+  groupName="qux"
+  groupValue="stadsdeel" // note that both `groupName` and `groupValue` are required for a named group to work
+  hasToggle
+  name="fooBarBaz"
+  options={[
+    { key: 'A', value: 'Centrum' },
+    { key: 'B', value: 'Westpoort' },
+  ]}
+  title="Stadsdeel"
+/>
+```
+
+will render:
+
+<img src="./toggle.png" alt="" width="220" />
```

Comment:
Looks the same...

Reply:
It does, yes. The HTML, though, is different.